tests/js/parser-server-tests.js

```diff
@@ -0,0 +1,155 @@
+/*
+
+  Test the parser by itself with as little interaction with the validator as possible (only static validator functions used).
+
+  Client-side tests and integration are covered by the validator tests, which make use of the parser internally.
+
+ */
+"use strict";
+var fluid  = fluid || require("infusion");
+var gpii   = fluid.registerNamespace("gpii");
+var jqUnit = require("jqUnit");
```

Comment:
Yes, I had to do this throughout to get everything working in the post-dedupe infusion world.